Project.toml

```
@@ -1,6 +1,6 @@
 name = "Ferrite"
 uuid = "c061ca5d-56c9-439f-9c0e-210fe06d3992"
-version = "0.3.14"
+version = "0.3.15"
```

Maybe
```suggestion
version = "0.4.0-DEV"
```
then? Next version will be 0.4.0.